src/me/components/Resources.tsx

```diff
@@ -59,6 +60,7 @@ const ResourceLists: FC = () => {
             ctaText="Learn more"
             title="New time-series engine for InfluxDB"
           />
+          <BlogFeed />
         </AnnouncementCenter>
       ) : (
         <DocSearchWidget />
```

Comment:
Also, would you mind getting rid of the default export? 😄

Reply:
done